scripts/prepare-machine.ps1

```diff
@@ -40,6 +40,26 @@ if ($IsWindows) {
         reg.exe add $TlsClientKeyPath /v DisabledByDefault /t REG_DWORD /d 0 /f | Out-Null
         reg.exe add $TlsClientKeyPath /v Enabled /t REG_DWORD /d 1 /f | Out-Null
     }
+    if ($Configuration -eq "Test") {
+        # Install OpenCppCoverage on test machines
+
+        # Install procdump on Windows if not already present.
+        if (!(Test-Path "C:\Program Files\OpenCppCoverage\OpenCppCoverage.exe")) {
+
+            Write-Host "[$(Get-Date)] Installing OpenCppCoverage..."
+
+            # Download the zip file.
+            $ExeFile = Join-Path $Env:TEMP "OpenCppCoverageSetup-x64-0.9.9.0.exe"
```

Comment:
Will a 64 bit copy of OpenCppCoverage work with 32 bit binaries? 

We should also add a check here either to not allow this if running a 32 bit os, or be smart enough to download the 32 bit copy on a 32 bit os.

Reply:
Before calling Invoke-WebRequest, add the following

`$ProgressPreference = 'SilentlyContinue'`

That will massively speed up the download. I need to do the same thing to the file copies in the perf script.

Reply:
To the first, yes it will work with 32-bit binaries.  However, that is a good point to ensure that this is a 64-bit OS, and possibly grab the 32-bit flavor in case of a 32-bit OS and test binary.